test/test.ml

```diff
@@ -1,23 +1,20 @@
-type ava
-type exe
+type t
 
-external t:ava = "ava" [@@bs.module]
+external ava:< 
+test : string -> (t -> unit) -> unit[@bs.meth]
+> Js.t = "ava" [@@bs.module]
 
-(*external ava: ava = "ava" [@@bs.module];;*)
+let test = fun run -> ava##test "" run
```

Comment:
Do we really need this? It seems you'll usually want to give it a name, and the more common case ought to be the shorter. If you really don't want to give it a name, you might as well just do `test "" ..` yourself.